Features/Authentication/UsersLogin.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Basic user with different roles login tests
+
+    @javascript
+    Scenario: As a Editor User, I want login to PlatformUI
+        Given I am logged in as an 'Editor' in PlatformUI
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a Member User, I want login to PlatformUI
+        Given I am logged in as an 'Member' in PlatformUI
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a Partner User, I want login to PlatformUI
+        Given I am logged in as an 'Partner' in PlatformUI
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a Subscriber User, I want login to PlatformUI
+        Given I am logged in as an 'Subscriber' in PlatformUI
+        Then I should see "Welcome to eZ Platform" title
```

Comment:
@andrerom I made these tests myself, to test the Issue because that is what the fix does it makes possible all the roles login.